package.json

```diff
@@ -4,9 +4,13 @@
   "description": "Watches package repo RSS feeds (e.g. PyPi and Maven) and pushes incoming changes to a RabbitMQ event queue",
   "main": "src/server.js",
   "scripts": {
+    "docker:build": "docker-compose build",
+    "docker:start": "docker-compose up rediscluster rabbitmq",
+    "docker:stop": "docker-compose down",
     "lint": "eslint .",
     "start": "node src/server.js",
-    "test": "ava"
+    "test": "ava test",
```

Comment:
You should having this command be `docker-compose run test; npm run docker:stop`. That way we default to running tests in docker.